3567/1/html_parser.rb

```diff
@@ -0,0 +1,30 @@
+require 'open-uri'
+require 'nokogiri'
+
+class HTMLParser
```

Comment:
Please add initialize, and move all methods which you use only in this class in private block